.github/CODEOWNERS

```diff
@@ -20,3 +20,5 @@
 /scripts/ @ovh/su-digital-tools-core-service-delivery
 /scripts/start-application.js @ovh/team-manager-dev
 /scripts/run-playwright.js @ovh/team-manager-transfo
+/scripts/run-playwright-bdd.js @ovh/team-manager-transfo
+/scripts/.env.development @ovh/team-manager-transfo
```

Comment:
Duplicates https://github.com/ovh/manager/pull/10924#discussion_r1506295008

Reply:
it's not related to the whole app, just to the test